File: Flow Control/Python/flow.py
def main():
    x = 10
    arr = [1,2,3,4,5]

    print("For loop:")
    for item in arr:
        print(item)

    print("While loop:")
    iterator = 0
    while iterator < 5:
        print(iterator)
        iterator += 1

    print("Switch case:")
    cases = {
        0: "X = 0",
        10: "X = 10",
    }
    print(cases.get(x, "X is not equal either to 0 or 10"))

main()

File: Flow Control/Javascript/flow.js
function main() {
  const x = 10;
  let arr = [1, 2, 3, 4, 5];

  if (x > 0) {
    console.log("X > 0");
  } else {
    console.log("X <= 0");
  }

  console.log("\nFor loop:");
  for (let i = 0; i < 5; i++) {
    console.log(arr[i]);
  }

  console.log("\nWhile loop:");
  let iterator = 0;
  while (iterator < 5) {
    console.log(iterator);
    iterator++;
  }

  console.log("\nSwitch case:");
  switch (x) {
    case 0:
      console.log("X = 0");
      break;
    case 10:
      console.log("X = 10");
      break;
    default:
      console.log("X is not equal either to 0 or 10");
      break;
  }
}

main();
